studio/pages/sign-in-fly-tos.tsx

```diff
@@ -0,0 +1,71 @@
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import Image from 'next/image'
+import { Button } from 'ui'
+import { useTheme } from 'next-themes'
+import { API_URL } from 'lib/constants'
+
+const SignInFlyTos = () => {
+  const router = useRouter()
+  const {
+    query: { extension_id, organization_id },
+  } = router
+  const { resolvedTheme } = useTheme()
+
+  const onSignInWithFly = async () => {
+    window.location.href = extension_id
```

Comment:
must it be a window.location.href? will a `<Link />` work?

Reply:
as we need to redirect to a external url (management api) to trigger sso flow. I'm not sure how to do it with <Link />

Reply:
`<Link />` also works with ext url

so something like `<Link href={`${API_URL}/tos/fly?extension_id=${extension_id}`}>...</Link>`
similar to how we link to docs site in the dashboard - but just need to double check though that it works. I'm okay with window.location.href otherwise if Link fit the use case here

Reply:
I will stick with Button for now cos it's easier to manage enable/disable with button.